app/views/admin/corporate_information_pages/index.html.erb

```diff
@@ -35,27 +32,30 @@
         text: "Updated by"
       },
       {
-        text: "Updated"
+        text: "State"
       },
       {
-        text: "State"
+        text: tag.span("State", class: "govuk-visually-hidden")
       }
     ],
     rows:
       @filter.editions.map do |edition|
         [
            {
-             text: render('title_row', edition: edition)
+             text: index_table_title_row(edition)
            },
            {
-             text: linked_author(edition.last_author, class: "govuk-link")
+             text: tag.p("#{time_ago_in_words edition.updated_at} ago", class: "govuk-!-margin-0") +
+                    "by " +
+                    linked_author(edition.last_author, class: 'govuk-link')
            },
            {
-             text: "#{time_ago_in_words edition.updated_at} ago",
+             text: index_table_state_row(edition),
            },
            {
-             text: admin_edition_state_text(edition),
-           }
+             text: sanitize("<a class='govuk-link' href='#{admin_edition_path(edition)}'>Edit <span class='govuk-visually-hidden'> #{edition.title}</span></a>"),
```

Comment:
can we not do this using the `link_to` helper?

Reply:
something like this maybe (untested)

```ruby
link_to sanitize("Edit <span class='govuk-visually-hidden'> #{edition.title}</span>", admin_edition_path(edition), class: "govuk-link"
```

Reply:
good shout have done 👍